modules/pam_env/tst-pam_env-retval.c

```diff
@@ -33,6 +33,10 @@ static const char usr_conf[] = TEST_NAME_DIR VENDOR_SCONFIGDIR "/pam_env.conf";
 
 static struct pam_conv conv;
 
+#ifndef PATH_MAX
+# define PATH_MAX 4096
+#endif
```

Comment:
Maybe it could be added to test_assert.h so it is not necessary to repeat it everywhere?

Reply:
yeah, that was another possibility, and I wanted to send at least a version of fixes; changed, thanks